src/commands/CommandType.h

```diff
@@ -44,21 +44,21 @@ class wxString;
 class AUDACITY_DLL_API OldStyleCommandType : public AudacityCommand
 {
 private:
-   ComponentInterfaceSymbol mSymbol;
+   mutable ComponentInterfaceSymbol mSymbol;
```

Comment:
The first commit in this pull request fails to build for me.  In this case, these classes are the problem:

AudioUnitEffects
AudioUnitEffectsModule
ApplicationPrefs

The last of these is not macOS specific -- and that explains why your PR is failing to build on all platforms.

The last of these is conditionally compiled, only if you enable check-for-updates in your build.  It is off by default for anyone who downloads and builds Audacity.  It is on for our Continuous Integration (CI) builds.

To turn it on for yourself, you must re-configure the project, setting certain CMake variables.  Re-configure just once, and those variable changes will be cached and reused with each build.

You can fix it using the cmake-gui program if you like, or do this just from a command line by passing `-Dname=value` arguments to cmake.

You need
audacity_has_updates_check=YES
audacity_has_networking=YES

Reply:
Thanks, that worked